src/meta/src/hummock/manager/mod.rs

```diff
@@ -967,18 +961,25 @@ where
 
             current_version.apply_version_delta(&version_delta);
 
+            let should_pause_write = self.calc_should_pause_write(current_version).await?;
```

Comment:
Use version update as ticker is incorrect. Because after write stall, version update may never happen.

Reply:
Refactored.